### PR TITLE
bump to release version

### PR DESCRIPTION
**Pull request by thehesiod**

### CODE REVIEW

**Comment by jettify** [comment]
👍

**Comment by thehesiod** [comment]
adding readme for how to upgrade botocore support

File: BOTOCORE_README.rst [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+works is by working backwards from `AioEndpoint._request`.  Because of this tight
+integration aiobotocore is typically version locked to a particular release of
+botocore.
+
+How to upgrade botocore
+-------------
+aiobotocore's file names try to match the botocore files they functionally match.
+For the most part botocore classes are sub-classed with the majority of the
+botocore calls eventually called...however certain methods like
+`PageIterator.next_page` had to be re-implemented so watch for changes in those
```
Comment (jettify) [inline BOTOCORE_README.rst]: By the way, in python 3.6 it is possible to use `yield` inside coroutine, so hopefully that huge iterator could be a bit simplified.

Comment (thehesiod) [inline BOTOCORE_README.rst]: I added this file because the last guy who tried updating support forgot to merge in a few changes

Comment (jettify) [inline BOTOCORE_README.rst]: good point

**Pull request merged**